server/pages/index.jade

```diff
@@ -44,16 +44,34 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		link(rel='apple-touch-icon', sizes='180x180', href='//s1.wp.com/i/favicons/apple-touch-icon-180x180.png')
 		link(rel='profile', href='http://gmpg.org/xfn/11')
 		link(rel='manifest', href='/calypso/manifest.json')
-		link(rel='stylesheet', href='https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese')
-		link(rel='stylesheet', href='//s1.wp.com/i/noticons/noticons.css?v=20150727')
-		link(rel='stylesheet', href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')
+
+		if !! renderedLayout
+			//- If we're server-side rendering, just load the stylesheets normally
+			- cssLinkAttrs = { rel: 'stylesheet' }
+		else
+			//- If we're not server-side rendering, load CSS async
+			- cssLinkAttrs = { rel: 'preload', as: 'style', onload: "this.rel='stylesheet'" }
+
+		link(href='https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese')&attributes( cssLinkAttrs )
+		link(href='//s1.wp.com/i/noticons/noticons.css?v=20150727')&attributes( cssLinkAttrs )
+		link(href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')&attributes( cssLinkAttrs )
+
+		if ! renderedLayout
```

Comment:
I think it's clearer to put this above, where the `cssLinkAttrs` variable is defined, so you only have 1 `if ... else` branch. Specially for the `preload` stuff, if you define `cssLinkAttrs={rel:'preload'}` and immediately after that include a polyfill, it's much clearer.